grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/Analytics.java

```diff
@@ -284,6 +278,30 @@ public long count() {
     }
 
     /**
+     * Compute the shortest path between two vertices.
```

Comment:
I think explicitly state in the java doc that you return a single shortest path and that there may be more.